contract/src/msgs/data_requests/sudo/remove_requests.rs

```diff
@@ -43,67 +43,78 @@ fn remove_request(
     let mut dr_escrow = DR_ESCROW.load(deps.storage, &dr_id)?;
 
     // add 1 so we can account for the refund message that may be sent
-    let mut bank_messages = Vec::with_capacity(messages.messages.len() + 1);
-    for message in messages.messages {
-        match &message.kind {
-            DistributionKind::Burn(distribution_burn) => {
-                dr_escrow.amount = dr_escrow.amount.checked_sub(distribution_burn.amount)?;
+    let mut bank_messages = Vec::new();
+
+    // We need to send messages in the order given, which should be: burn -> data_proxy_reward -> executor_reward
```

Comment:
If we do not enforce ordering, is better to keep it out
```suggestion
    // We need to send messages in the order given
```

Reply:
Do you mean if contract side doesn't enforce it?

Reply:
Yeah I think that's it. Basically to prevent confusion if the logic ever changes on the chain side and we forget to update the comment here.